src/Images.jsx

```diff
@@ -73,7 +77,7 @@ class Images extends React.Component {
                     let error = (
                         <React.Fragment>
                             <strong>
-                                {cockpit.format(_("Failed to download image $0"), imageName)}
+                                {cockpit.format(_("Failed to download image $0:$1"), imageName, imageTag || "latest")}
```

Comment:
Is no tag equivalent to `:latest` ?

Reply:
Right, if you don't specify tag podman will pull latest tag by default.
From `man podman pull`:
`If an image tag is not specified, podman pull defaults to the image with the latest tag (if it exists) and pulls it.`

Reply:
cool! I'm good with this then :)